src/Runner.php

```diff
@@ -31,17 +29,20 @@ public static function program()
 
         // Init. the Gource class with default properties
         $gourceClass = new Gource();
-            $gourceClass->setResolution('1920x1080');
+            $gourceClass->setResolution('1920x1080')
+                ->setStartDate('2017-10-01');
 
         // Init. AutoGourcer with default settings.
         $ag = new AutoGourcer();
         $ag->setGit($gitClass)
-            ->setRepoCount(10)
+            ->setRepoCount(5)
```

Comment:
What is the significance of 5 here? Is this a default setting? Can it get overridden anywhere?

Reply:
Correct, 5 is the default. Yes it can be overwritten by the end user if they implement their own initiation logic.